etc/orthography.tsv

```diff
@@ -0,0 +1,88 @@
+Grapheme	IPA	EXAMPLES
+^	NULL	0
+$	NULL	0
+-	NULL	wai-oot;i-thlung-a;i-thlung-a;i-hling-a;i-hling-a
+´	NULL	pu-kwā´-num;ka-tit´-too;in´-to-op;noo-ka-tam´-tha;is´-hail-o-wih
+’	NULL	shi-li-kit-tim-win-’go;ni-’ko’k-a-hik;ni-’ko’k-a-hik;tsoo-wan-kit-tim-win-’go;spl-’kein
+gw	gʷ	duha-ho-lo-gwodi;tsel´-e-de-gwus;stō-le-gwan;goo-gwilla;goo-gwilla
+hw	hʷ	hahwi-e;koos-hwa;koos-hwa;koos-hwa;kla-le-tla-hwlth
+lh	ʎ	til-koolh-im-yoot;til-koolh-im-un-nah;slin-ealh-kum-ealh;kulh-lātik;nish-ai-ulh-slannē
+ll	ɬ	ikalla;ska-kullā;nis-tallis;nis-tallis;kwullā
+sh	ʃ	kash-kash;shin-shin-ashi;shin-shin-ashi;shish-ootum;kash-kash
+wh	wʰ	tsi-hi-a-whatl;kus-whun-ook;kus-whun-ook;hun-whun-ookwi;is´-whuh-sē-whult
+ōh	ō ʔ	slanai-ōhl;kōht
+^āh	ā ʔ	āh-alph
+^īh	ī ʔ	īha-kommitz;īh-ā;īh-ā;īhl-thlē;īh-kwīn
+hs$	ʔ s̩	thluhs;eiloohs;luhs;poot-umit-kwo-luhs
+tsh	tʃʰ	tloo-tshimun;sē-tshin-kooi;si-asē-tshuk-oap;klootsh-timtz;te-tshā
+wh$	w ʔ	hul-lawh;slawh;yoawh;ow-awh;īdawh
+zh$	z ʔ	kwai-el-izh
+āh$	ā ʔ	hun-nāh;un-nāh;ni-lāh;hāh;hāh
+ēh$	ē ʔ	kuh-kwēh;tshin-kat-ka-tēh
+ĕh$	ĕ ʔ	skil-a-kĕh
+īh$	ī χ	shīh;hut-līh;hut-līh;sam-tlīh
+ōh$	ō ʔ	tin-ā-ōh;tlōh;kōh;mōh;mōh
+a	a	wai-oot;swaika;soi-kan;kah-tin;ikalla
+ā	ā	pu-kwā-num;pu-kwā´-num;pook-wā-num;stil-tāmsh;shā-wut
+ă	ă	sĭttăk;sātsă-tăkum;sātsă-tăkum;kslă-koo-num;kslă-koo-num
+ts	ts	tsa-tsa-ta-kum;sātsă-tăkum;tloo-tsi-tsi;tloo-tsi-tsi;tats-guika
+e	e	kewlh;ske-ki-thlil;akag-eluk;tā-gool-taine-tīn;tse-ke
+ē	ē	swē-ka;sim-ēm;to-kwunē-e;tē-nā-se;kloh-tēlt
+ĕ	ĕ	tsalĕ-gedit;a-ko-ko-ut´-tĕ-na;in-kĕ-shēl´-ni-kuk;snĕt;sumĕ-hai-it-tshin
+ee	e e	keet;keet;keet;skee-ool-ki
+x	x	nux-soo;nux-soo;nux-soo;nux-soo;tsim-aux-shin
+χ	χ	ki-χi;kitl-χow
+x$	x	spsis-sax;ko-kwinix;tla-la-hlam´-ix;moax;moax
+h	h	i-hling-a;ē-hlin;kah-tin;sahlt;pahtl-ke
+^h	h	hun-nāh;hā-tlel-a;hā-puh;ha-push;ha-push
+h$	h	youh;in-tlah;skul-tum-ih;hā-puh;til-koolh-im-un-nah
+i	i	wai-oot;i-hling-a;ē-hlin;enika;swaika
+ī	ī	wīns;tā-gool-taine-tīn;kī-ka;tshīl;tshīl
+ĭ	ĭ	sĭttăk;smoo-ītlĭts;mā-kĭn;kuai-ĭgoot;toa-whĭ-ai
+j	j	n-jada;ti-koot-jung-a;ti-koot-jung-a;di-jasha;ti-jas-ka
+g	g	i-thlung-a;i-hling-a;akag-eluk;tā-gool-taine-tīn;n-gunna
+k	k	pook-wā-num;enika;swaika;soi-kan;swē-ka
+gh	gʰ	ti-ow-gha;ti-tla-ghă;uh-hi-ghutti;uh-hi-ghutti;shā-wut-uh-ghutti
+gs	g s	yungs-a-kīn
+k$	k	sĭttăk;akag-eluk;’kl-wām-ilk;walak;kl-ku-am-ilk
+kh	kʰ	’kha;skh-ai-oos;ka-kloo-khluh´-ā;uh-a-kha-wo;’khai
+ks	k s	salks-māna;salks-māna;sa-ykso;tahks-ta;muksn
+kw	kʷ	pu-kwā-num;pu-kwā´-num;to-kwunē-e;a-poo-kwunse;ani-kwul
+kwh	kʷʰ	ut-la-kwhl
+kws	kʷ s	is-kws-sum-ēm´;hakws-smuk-wătshi
+l	l	i-thlung-a;i-hling-a;ē-hlin;stil-tāmsh;im-ilkh
+m	m	im-ilkh;skul-tum-ih;tloo-tshimun;smoo-tlats;sim-ēm
+ms	m s	timst;timst
+mh$	m h	et-shumh;kwalimh
```

Comment:
really? usually `mh` is referring to some single sound, and can you pronounce `mh` in the end of a word? And did you not say `h` is a glottal stop, above?